bundles/org.openhab.binding.velbus/src/main/resources/OH-INF/config/config.xml

```diff
@@ -896,4 +921,678 @@
 			<advanced>true</advanced>
 		</parameter>
 	</config-description>
+
+	<config-description uri="thing-type:velbus:81channelDevice">
+		<parameter name="address" type="text" required="true">
+			<label>Address (Channel A0...7)</label>
+			<description>The velbus address of the device</description>
+		</parameter>
+		<parameter name="subaddress1" type="text" required="true">
+			<label>Channel A8...15</label>
```

Comment:
```suggestion
			<label>Address (Channel A8...15)</label>
```

Otherwise the label isn’t self-explanatory.

Reply:
Updated in last commit.